### PR TITLE
Create dummy pages

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+@url.route('/')
+def index():
+    return render_template('index.html')
+
+
+@url.route('/<query>')
+def print_user_input(query):
+    if '=' in query:
+        query_container, query_value = query.split('=')
+        return 'Your query is {} which is equal to {}'.format(query_container, query_value)
```
Comment (mknecht) [inline app/views.py:14]: want to do something cool?

```
return 'Your query is {} which is equal to {}'.format(*query.split('='))
```

embrace the power of python @admiral96 ! :)

(also, that pair is usually called a key-value-pair :)

File: app/templates/base_error.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>{{ title if title else "Error"}}</title>
```
Comment (mknecht) [inline app/templates/base_error.html:4]: nice … but why not just hardcode it? it's not like you're using it (unless i missed it)

File: app/templates/base.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
+    {% if title %}
+    <title>{{ title }}</title>
+    {% else %}
+    <title>DRC Web-Hook</title>
```
Comment (mknecht) [inline app/templates/base.html:6]: same here

File: run.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#!flask/bin/python
+import sys
+from app import mulungwishi_app
+
+# This line is important so flake8 must ignore this one.
```
Comment (mknecht) [inline run.py]: yes but what does flake8 need to ignore? you're not explaining this for someone that neither sees the error, nor knows what they can and _cannot_ do with this line.

Comment (admiral96) [inline run.py]: It's in below this of line

File: app/views.py (hunk 1 above)
Comment (admiral96) [inline app/views.py:14]: I would like it to remain for readability purposes :) thoughts? @mknecht

Comment (mknecht) [inline app/views.py:14]: that's a good argument. let it as it is then :+1:

File: run.py (hunk 4 above)
Comment (mknecht) [inline run.py]: your comment needs to answer these questions:
1. which error are you silencing?
2. why are you silencing it

“it's important” answers neither :)

File: app/templates/base_error.html (hunk 2 above)
Comment (admiral96) [inline app/templates/base_error.html:4]: Yep. I used it for different title on different errors

Comment (mknecht) [inline app/templates/base_error.html:4]: okay

File: app/templates/index.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{% extends  "base.html" %}
+
+
+{% block content %}
+    <h1>DRC Web-Hook app is coming soon.</h1>
+    <br>
+    <h4>What is DRC Web-Hook app?</h4>
+    <p>It is an application that would receive data from farmers and responds based on the information by <b>Chris</b> that would make the life of farmers easy and more productive.</p>
+    <p>Please see also <a href="https://www.youtube.com/watch?v=qkBsuUHCFWo" target="_blank">Mulungwishi Fertilizer Project</a></p>
```
Comment (mknecht) [inline app/templates/index.html]: this is another card …

**Comment by mknecht** [comment]
Can you please paste in the PR description, which Mingle card you're solving here? Otherwise, reviewers can't really check the requirements.

**Comment by admiral96** [comment]
No card on mingle but necessary for card #3995

**Comment by mknecht** [comment]
@admiral96 nope, you're adding _tons_ of handlers, three for errors, one for success case … 75% of that is not needed for that card.

Merge this one, but be more precise next time please.

**Pull request closed without merging**